projects/gift-crowdfunding.md

```diff
@@ -11,6 +11,11 @@ This web app will allow users to build collaborative giftlists for each gifting
 
 ### Project Management Tool
 
-Taiga
+Taiga (Professors should have gotten an invite)
```

Comment:
No need to say all those things like "professors ..." :)

Feel free to update, I'll merge this "as is" for now.